### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by Xiaodingab** [neagle/gokibitz#169]
Can't see comments on Android tab. 4.1, Asus MeMO 8. The comments window does not display.

Thnx, love the site.

**Comment by levelonedev** [comment]
Can you reproduce this on an Android emulator?

**Comment by Xiaodingab** [comment]
No, all I have is my tab, sorry. I can send screen shots, if you like.

Thnx!

----- Original Message -----
